README.md

```diff
@@ -34,6 +34,9 @@ spec:
     provider: <GCP_OR_AWS>
     region: <AWS_REGION>
 ```
+Above configuration assumes that OpenEBS is installed in `openebs` namespace.
```

Comment:
This should appear in the above spec by default so that user knows that there's an option to configure.

Reply:
I haven't added it intentionally since user don't need to configure this parameter if openebs is installed in `openebs` namespace.